File: TEO/Mav.UI/Mav.Ui/src/app/models/admin/licence.view.model.ts
import { OrganisationViewModel } from "./organisation.view.model";

export class LicenceViewModel {

    public id: number;

    public organisationId: number;

    public maxCentres: number;

    public cost: number;

    public academicYear: number;

    public paid: boolean;

    public organisation: OrganisationViewModel;
}

File: TEO/Mav.Models/ViewModels/LicenceViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Mav.Models.ViewModels
{
    public class LicenceViewModel
    {
        public int ID { get; set; }

        public int OrganisationID { get; set; }

        public int MaxCentres { get; set; }

        public double Cost { get; set; }

        public int AcademicYear { get; set; }

        public bool Paid { get; set; }

        public OrganisationViewModel Organisation { get; set; }
    }
}
